parents.md

```diff
@@ -0,0 +1,50 @@
+# What is the WebDev Guild - A guide for parents
+
+Howdy, and welcome to the WebDev Guild! The WebDev Guild can be a great opportunity for your child to develop technical development skills that can benefit them now and in the future. Learning to code not only opens door for future career development, but can improve computational problem-solving skills in all areas. As your child sets out on this journey, we want you, as a parent, to be well-informed about the Guild, and your child’s learning.
+
+## What is the WebDev Guild?
+
+The WebDev Guild is a website programming club for kids. We provide resources and guides for students (we call them Guilders) to learn the basic principles of web development. The Guilders complete requirements and earn badges to showcase what they have accomplished. This isn't a watered-down program—the resources we provide are real-world guides created by industry leaders. Our goal is to develop Guilders who are able to assemble fully functional web apps by the time they complete all of the badges and requirements.
+
+Guilders are paired with experienced adult mentors who can answer their questions, guide their learning, and pass off their requirements.
+
+The Guild also has regular meetings where Guilders meet to discuss their progress, show off the projects they are working on, and pass off badge requirements.
+
+## Is there anything I should be worried about?
+
+As with anything on the internet, there are a few risks with being involved in the WebDev Guild:
+
+-	Programmers sometimes use saucy language in their blog posts and guides. Since many of the badges encourage Guilders to do their own searching, they might come across this language.
+-	As with any interactions on the web, Cyberbullying is always a threat—both from inside and outside the Guild.  We encourage any Guilder who experiences bullying to speak up—either to their parent, or to a Guild leader.
+-	To develop and deploy web applications, students will need to create accounts on trusted, secure websites. However, any web activity also comes with a risk that personal information could be discovered and exploited. 
+
+We take these matters very seriously and employ the following measures to combat them:
+
+-	We ensure that all of our badges link to posts and guides of the highest quality. If we find that these guides are not up to our standards, we will remove them immediately.
+-	We will not tolerate cyberbullying of any kind. If we receive reports of cyberbullying, we will immediately suspend the offender's Discord account and take appropriate action, up to and including ejection from the guild.
```

Comment:
We may be able to remove them from the server, but they will still have access to direct messages to the person.